framework/src/auxkernels/FluxAux.C

```diff
@@ -0,0 +1,42 @@
+#include "FluxAux.h"
```

Comment:
You'll need the MOOSE headers at the top of all source and header files. Copy from another file in this directory.